### DISCUSSION TITLE
Replacement unicode character in text extraction

### DISCUSSION THREAD

**Discussion by indigoviolet** [pymupdf/PyMuPDF#2952] [Q&A]
[test.pdf](https://github.com/pymupdf/PyMuPDF/files/13806565/test.pdf)

```shell
❯ mutool draw -F text test.pdf | head    
����������������� ������ ����
NYPA2514    12/06/21

�������������������
���������������������������������

��
����������������

������
```

And of course, PyMuPDF also has 65333s

```shell
❯ pdftotext test.pdf - | head
NYPA2514

Partner's share of
20

modifications (see instructions)

State additions
A ' Total amount
```

pdfplumber also parses this file successfully. 

Any suggestions?

**Comment by JorjMcKie** [comment]
This seems to be an MuPDF issue - although there are some unorthodox font specifications (missing `/Encoding` keys).
Ok to forward this to MuPDF's issue system?

**Reply by indigoviolet** [reply]
Yes, thank you.
